### PR TITLE
Advisors

### PR DESCRIPTION
**Pull request by MostafaBalata**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# dependencies
```
Comment (krzkaczor) [inline .gitignore]: can you remove not important files? i think we only need to ignore few directories (node modules, dist)

File: page/index.ejs [hunk 2]
```diff
@@ -9,6 +9,8 @@
             <div id="react-root"></div>
 
             <script src="/main.js"></script>
+            <%- include('partials/advisors'); %>
```
Comment (krzkaczor) [inline page/index.ejs]: lets include it before `main.js`

File: page/ts/index.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const $ = require('jquery');
```
Comment (krzkaczor) [inline page/ts/index.ts]: use `import * as $ from 'jquery'`

File: page/ts/index.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import 'owl.carousel';
+
+$(document).ready(function(){
+    // $('.has-carousel').show();
```
Comment (krzkaczor) [inline page/ts/index.ts]: remove commented code

File: webpack.config.js [hunk 5]
```diff
@@ -1,11 +1,12 @@
 const path = require('path');
 const webpack = require('webpack');
+const jquery = require('jquery');
```
Comment (krzkaczor) [inline webpack.config.js]: do you need that import here?

Comment (krzkaczor) [inline webpack.config.js]: I think it's not used anywhere.

File: page/sass/pictures.scss [hunk 6]
```diff
@@ -0,0 +1,9 @@
+img.rounded-image{
```
Comment (krzkaczor) [inline page/sass/pictures.scss]: duplicated selectors? Just merge these two.

File: page/sass/advisors.scss [hunk 7]
```diff
@@ -0,0 +1,47 @@
+
+.advisors p{
```
Comment (krzkaczor) [inline page/sass/advisors.scss]: use sass features to reduce repeated code: 
```
.advisors {
    p{ ...}
    h3 { ... }
}
```

http://sass-lang.com/guide -> nesting

File: page/partials/advisors.ejs [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+                    <div class="col-md-6">
+                        <div class="person-info">
+                            <img class="rounded-image" src="../assets/mostafabalatabw.jpg"/>
+                            <div class="person-details">
+                                <h4 class="name"><a href="#" >Micheal Jackson</a></h4>
+                                <h4 >Mangrove Capital</h4>
+                            </div>
+                        </div>
+                    </div>
+                </div><!-- End row -->
```
Comment (krzkaczor) [inline page/partials/advisors.ejs]: do not create new rows. Just put all images in one row and let css figure out when to break to new line. Then it's responsive.  Read: https://stackoverflow.com/a/31049088/580181

File: page/ts/index.ts (hunk 3 above)
Comment (MostafaBalata) [inline page/ts/index.ts]: This is not working with me.

File: page/ts/index.ts [hunk 9]
```diff
@@ -1,8 +1,8 @@
-const $ = require('jquery');
+// const $ = require('jquery');
```
Comment (krzkaczor) [inline page/ts/index.ts]: remove commented code

File: page/partials/advisors.ejs [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+                        </div>
+                    </div> <!--end advisor -->
+
+
+                </div> <!-- end carousel -->
+                    
+            </div>
+            <div class="col-md-7">
+
+                <div class="row person-info-container">
```
Comment (krzkaczor) [inline page/partials/advisors.ejs]: `person-info-container` is used two times but in different contexts. Probably you dont need it here. If you need it on line 39. If you need that margin maybe add it to `advisors` class.

File: page/partials/advisors.ejs [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+                    <div class="col-md-6 person-info-container">
```
Comment (krzkaczor) [inline page/partials/advisors.ejs]: `person-info-container` will be reused on team section as well. Move all related styles to separate file.

**Pull request merged**